test.py

```diff
@@ -158,7 +158,7 @@ def generate_file_pairs(
             cmd = (
                 ["sgrep-lint"]
                 + extra_args
-                + ["--strict", "--json", "--no-rewrite-rule-ids", "-f", str(filename)]
+                + ["--strict", "--strict-parsing", "--json", "--no-rewrite-rule-ids", "-f", str(filename)]
```

Comment:
what does this change?

Reply:
If sgrep has a parse error on a code file, under this flag it will fail rather than silently skipping that file. This means our test suite will now fail if someone tries to commit a file sgrep can't parse to this repo.